modules/kubernetes-addons/aws-ebs-csi-driver/main.tf

```diff
@@ -45,3 +48,45 @@ resource "aws_iam_policy" "aws_ebs_csi_driver" {
     try(var.addon_config.tags, {})
   )
 }
+
+module "helm_addon" {
+  source = "../helm-addon"
+  count  = var.enable_self_managed_aws_ebs_csi_driver && !var.enable_amazon_eks_aws_ebs_csi_driver ? 1 : 0
+
+  helm_config = merge({
+    name        = local.name
+    description = "The Amazon Elastic Block Store Container Storage Interface (CSI) Driver provides a CSI interface used by Container Orchestrators to manage the lifecycle of Amazon EBS volumes."
+    chart       = local.name
```

Comment:
this should be a string of `"aws-ebs-csi-driver"` because the chart name is what is defined externally and cannot be changed when the `name` changes